doc/setup.md

```diff
@@ -115,28 +145,22 @@ When developing with SkyPortal on mac, you may  also need to configure your shel
 ```
 nano ~/.zshrc
 ```
-#### Set enviroment variables to their installation paths
-After installing the libraries with Homebrew, you'll need to set environment variables to their installation paths. Replace the placeholder text `<path-to-library>/<version>` with the actual path that Homebrew provides upon sucessful installation.
+#### Set environment variables to their installation paths
+After installing the libraries with Homebrew, you'll need to set environment variables to their installation paths. Replace the placeholder text `<path-to-library>/<version>` with the actual path that Homebrew provides upon successful installation.
```

Comment:
here there is a mention of <version> but it's not in the actual template underneath, can you add it to the template? thanks